modules/build/src/test/scala/scala/build/tests/SourcesTests.scala

```diff
@@ -69,44 +69,46 @@ class SourcesTests extends munit.FunSuite {
       val scopedSources = crossSources.scopedSources(BuildOptions()).orThrow
       val sources       = scopedSources.sources(Scope.Main, BuildOptions())
 
-      expect(sources.buildOptions.classPathOptions.extraDependencies.map(_.value) == expectedDeps)
+      expect(sources.buildOptions.classPathOptions.extraDependencies.map(
+        _.value
+      ).toSet == expectedDeps.toSet)
       expect(sources.paths.isEmpty)
       expect(sources.inMemory.length == 1)
       expect(sources.inMemory.map(_._2) == Seq(os.rel / "something.scala"))
     }
   }
 
-  test("dependencies in .scala - using witin tests") {
-    val testInputs = TestInputs(
-      os.rel / "something.scala" ->
-        """using target test
-          |using libs "org1:name1:1.1" "org2::name2:2.2"
-          |using lib "org3:::name3:3.3"
-          |import scala.collection.mutable
-          |
-          |object Something {
-          |  def a = 1
-          |}
-          |""".stripMargin
-    )
-    val expectedDeps = Nil
-    testInputs.withInputs { (_, inputs) =>
-      val crossSources =
-        CrossSources.forInputs(inputs, Sources.defaultPreprocessors(CustomCodeWrapper)).orThrow
-      val scopedSources = crossSources.scopedSources(BuildOptions()).orThrow
-      val sources       = scopedSources.sources(Scope.Main, BuildOptions())
-
-      expect(sources.buildOptions.classPathOptions.extraDependencies.map(_.value) == expectedDeps)
-      expect(sources.paths.isEmpty)
-      expect(sources.inMemory.length == 0)
-    }
-  }
+//  test("dependencies in .scala - using witin tests") {
```

Comment:
Does that mean that there's no equivalent to `using target test` syntax?

Reply:
It's re-enabled in further commits